pkg/cni/chained.go

```diff
@@ -124,6 +124,7 @@ func insertCNIConfig(oldconfig []byte) ([]byte, error) {
 	// add kmesh-cni configuration
 	kmeshConfig["type"] = kmeshCniPluginName
 	kmeshConfig["kubeConfig"] = kubeconfigFilepath
+	kmeshConfig["mode"] = mode // provide mode here, so that kmesh-cni can decide how to run
```

Comment:
I think it's better for the comment to explain the two modes that are now supported